.github/workflows/audit.yml

```diff
@@ -25,6 +25,36 @@ jobs:
         distribution: 'zulu'
         java-version: '17'
 
+    - name: Install Python
+      uses: actions/setup-python@v5
+      with:
+        # Fivetran build script requires Python 3.10, not above.
+        # Testing the destination found that it also requires 3.10, latest does not work.
+        python-version: '3.10.8'
+
+    - name: Install Python dependencies
+      run: python -m pip install --upgrade pip setuptools wheel
+
+    - name: Install Python connector requirements
+      working-directory: examples/connector/python
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt 
+
+    - name: Run Python connector build script
+      working-directory: examples/connector/python
+      run: ./build.sh
+
+    - name: Install Python destination requirements
+      working-directory: examples/destination/python
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
I think it is enough to upgrade pip once. Even more `actions/setup-python@v5` installs the `pip`. Maybe not the latest version but enough to install the packages. And github-hosted agents also have `pip` installed.